src/pages/staticPages/tools/oncoprinter/OncoprinterGeneticUtils.ts

```diff
@@ -567,17 +614,46 @@ export function parseGeneticInput(input:string):{status:"complete", result:Oncop
                             throw new Error(`${errorPrefix}Alteration "${alteration}" is not valid - it must be "HIGH" or "LOW" if Type is "PROT"`);
                         }
                         break;
+                    case "fusion":
```

Comment:
is handling fusion part of scope of PR?

Reply:
no, I just have to handle it separately now that I am parsing mutations differently due to the _GERMLINE and _DRIVER, whereas before I was handling it together with other mutations.